### ISSUE TITLE
Questions about the data

### ISSUE THREAD

**Issue by joakimjohnander** [JonathonLuiten/TrackEval#41]
Hey,

First and foremost, thank you for this work to facilitate analysis of multiobject trackers. After having a look at the readme-files, code, and provided data I have a couple questions.
i) The RobMOTS data, train_data.zip, seems to contain detections, ground-truth, and the predictions made by the simplest possible tracker. The detections are missing on BDD100k and waymo. The ground-truth seems to be missing for waymo. Are the missing detections and ground-truth incoming?
ii) Are the corresponding val_data.zip and test_data.zip missing? Will at least the validation set be provided with ground-truth?
iii) I noticed that there are no corresponding images provided. For the RobMOTS challenge, are we expected to rely only on the detections? For tracker evaluation and analysis in general, do you plan to supply also matching images?
iv) On that note, it seems straightforward to download images myself and match them to the provided detections, as the detection files are named after the sequence names. However, how do I match the image frames to the time steps? Specifically, the YouTubeVIS detection times do not seem to correspond to the YouTubeVIS dataset times.
v) Both the TAO-dataset and BDD100k are utilized. Is this not problematic as TAO contains some sequences from BDD100k?

Best regards,
Joakim

**Comment by JonathonLuiten** [comment]
Hey! Thanks for your interest.

Here is the answers to your questions.

i) BDD-100K train is coming (very soon). Waymo train is not (this was too large for us to process and provide it, feel free to use it otherwise if you wish).

ii) Supplied detections for val and test are coming. GT will NOT be supplied for either, however and 'upload server' will be supplied for both. 

iii) The images are provided, see data download link on this page: https://eval.vision.rwth-aachen.de/rvsu-workshop21/?page_id=110

iv) Just use the images we supply on the link in (iii)

v) There may be some overlap here, but there also quite likely isn't (Both TAO and BDD100k-MOTS are both only small subsets of all of the much larger BDD-100k videos). I haven't analyzed this myself if there is or isn't overlap, but it's not something I'm particularly worried about.

Hope that answers your questions!

Good luck with the challenge!

Jono

**Comment by joakimjohnander** [comment]
Great, thanks for the swift reply.

**Comment by JonathonLuiten** [comment]
An update on your questions.

i) BDD detections now available.

ii) val + test detections now available

General: I have updated the readme [here](https://github.com/JonathonLuiten/TrackEval/tree/master/docs/RobMOTS-Official) so that hopefully the answer to your other questions are more obvious to the next person that reads it.

Also: There is now a lot of useful code in trackeval/baselines which you will probably find helpful.

Thanks again for your questions and interest in the challenge.

Jono

**Comment by joakimjohnander** [comment]
Cheers, thanks!
